sormas-api/src/main/java/de/symeda/sormas/api/importexport/ExportFacade.java

```diff
@@ -38,6 +38,8 @@
 
 	List<ExportConfigurationDto> getExportConfigurations(ExportConfigurationCriteria criteria);
 
+	List<ExportConfigurationDto> getPublicCustomExportConfigurations(ExportConfigurationCriteria criteria);
```

Comment:
There is no need to add a new method and duplicate the existing one for getting the public configurations. 

Just extend the criteria with an isPublic flag.